test/unit/connectors/CustomsDataStoreConnectorSpec.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.UnitSpec
+import config.AppConfig
+import org.scalatest.GivenWhenThen
+import org.scalatestplus.play.guice.GuiceOneAppPerSuite
+import play.api.test.Injecting
+import testdata.TestData
+
+class CustomsDataStoreConnectorSpec extends UnitSpec with GivenWhenThen with GuiceOneAppPerSuite with Injecting {
```

Comment:
Please don't use Guice application for unit tests. They are not really unit if you do so.

Reply:
In general I agree with you, but to test that AppConfig (and I do not want a fake instance here) provides to the connector the expected url I need `app` to inject the instance. I could move this unit test in the integration tests, if you prefer, but this would change nothing in terms of performance. WDYT?

Reply:
Refactored as Integration Test.